tests/unit/drive_controls_test.py

```diff
@@ -0,0 +1,34 @@
+from robot.rospackages.src.mcu_control.scripts.DriveControls import twist_to_rover_command, accelerate_value
+import robot.rospackages.src.mcu_control.scripts.DriveControls as dc
```

Comment:
Can't you flip these two lines and then do `from dc import ...`?

Reply:
nope, does not work